programs/autocrat_v0/src/lib.rs

```diff
@@ -457,19 +473,13 @@ pub struct InitializeDAO<'info> {
     #[account(
         init,
         payer = payer,
-        space = 8 + std::mem::size_of::<DAO>(),
-        // We will create a civilization of the Mind in Cyberspace. May it be
-        // more humane and fair than the world your governments have made before.
-        //  - John Perry Barlow, A Declaration of the Independence of Cyberspace
-        seeds = [b"WWCACOTMICMIBMHAFTTWYGHMB"], 
-        bump
+        space = 8 + std::mem::size_of::<DAO>()
     )]
     pub dao: Account<'info, DAO>,
     #[account(mut)]
     pub payer: Signer<'info>,
     pub system_program: Program<'info, System>,
-    #[account(mint::decimals = 9)]
-    pub meta_mint: Account<'info, Mint>,
+    pub token_mint: Account<'info, Mint>,
     #[account(mint::decimals = 6)]
     pub usdc_mint: Account<'info, Mint>,
```

Comment:
The pubkey of the USDC mint is known on mainnet and a cfg check could possibly be used to ensure that USDC is used and not another 6 decimals token. Not a massive deal though

Reply:
Yep, agreed. Let me create this as an issue.